package.json

```diff
@@ -23,8 +23,6 @@
   },
   "homepage": "https://github.com/andela/freyja-ah-frontend#readme",
   "dependencies": {
-    "babel-eslint": "^10.0.2",
-    "express": "^4.17.1",
     "@babel/polyfill": "^7.4.4",
     "@babel/runtime": "^7.5.4",
```

Comment:
@beejay1293 Both `@babel/runtime` and `@babel/polyfill` does not seem to be used anywhere in the project. Are you using it anywhere? If not kindly uninstall them. I reached out to @Ochowo And she confirmed they are both unused packages.